gallery/src/styles/NxCard/NxCardLayoutExample.tsx

```diff
@@ -0,0 +1,300 @@
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Content in columns</h3>
+          </header>
+          <div className="nx-card__content nx-card__content--columns">
+            <div className="nx-card__call-out">
+              100
+            </div>
+            <div className="nx-card__text">Data point details</div>
+          </div>
+          <footer className="nx-card__footer"><a href="#" className="nx-text-link">Link</a></footer>
+        </section>
+        <section className="nx-card" aria-label="Card with NxReadOnly">
+          <div className="nx-card__content">
+            <dl className="nx-read-only">
+              <dt className="nx-read-only__label">
+                Component Foo
+              </dt>
+              <dd className="nx-read-only__data">
+                Component Foo does not contain proprietary packages
+              </dd>
+              <dt className="nx-read-only__label">
+                Component Bar
+              </dt>
+              <dd className="nx-read-only__data">
+                Component Bar does not contain proprietary packages
+              </dd>
+            </dl>
+          </div>
+        </section>
+        <section className="nx-card" aria-label="Icon in callout">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Icon in callout</h3>
+          </header>
+          <div className="nx-card__content">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faShapes} className="nx-card__call-out-icon" />
+            </div>
+            <div className="nx-card__text">Data point details</div>
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+      </div>
+      <div className="nx-card-container">
+        <section className="nx-card" aria-label="Another card with icon in callout">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__content">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faShapes} className="nx-card__call-out-icon" />
+            </div>
+            <div className="nx-card__text">Data point details</div>
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+        <section className="nx-card" aria-label="Server card">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__content">
+            <div className="nx-status-indicator">
+              <span>Server one</span>
+            </div>
+            <div className="nx-status-indicator">
```

Comment:
Probably not a good idea to be using currently made-up but soon-to-be-real `nx-` classes in the example.

Reply:
> Probably not a good idea to be using currently made-up but soon-to-be-real `nx-` classes in the example.

I just converted them to `<p className="nx-p">`: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/6ec8ec9daf8d17faea5f865409f9ec997663e2bd